tests/eas/load_tracking.py

```diff
@@ -715,7 +724,26 @@ def _get_util_mean(self, start, end, df, cpus):
         util_mean = {}
         df = df[start:end]
         for cpu in cpus:
-            util = df[(df.cpu == cpu) & (df.path == '/')].util
+            util = df[(df.cpu == cpu) & (df.path == '/')]
+            for task in util['__comm'].unique():
+                if task != "<idle>" and not task in self.tasks_desc.keys():
+
+                    # The same task might appear under several pids, calculate
+                    # the total percentage of time for this task
+                    pids = util[(util['__comm'] == task)]['__pid'].unique()
+                    creep_time = 0
+                    for pid in pids:
+                        sched_assert = self.get_sched_assert_pid(experiment, pid)
+                        creep_time += sched_assert.getRuntime(window=(start, end), percent=True)
+
+                    # Make sure that any other processes that creeped into the
+                    # cpu we are testing on are insignificant
+                    msg = "task {} creeped into CPU{} for {}% during {}".format(task, cpu, creep_time, phase)
```

Comment:
`"... during Phase[{}]"` ?
... just to be explicit we are reporting a phase name?

Reply:
because phase is the unit of test. The tests are done in steps of phases. So we catch the noise during that step/phase and report if we caught an issue in that phase.